### DISCUSSION TITLE
将arthas打到镜像中报错了

### DISCUSSION THREAD

**Discussion by zhiliatom** [alibaba/arthas#2488] [Q&A]
Step 6/8 : COPY --from=hengyunabc/arthas:latest /opt/arthas /opt/arthas
Unknown flag: from

使用命令将arthas打入容器中，报错了
[root@localhost console]# docker version
Client:
 Version:         1.13.1
 API version:     1.26

**Comment by hengyunabc** [comment]
不行就试下 wget 下载，再 unzip，差不多的。
